### PR TITLE
New major version for Flow 3.0

### PR DESCRIPTION
**Pull request by tomivirkki**
Drops Bower + HTML imports

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
 
     <profiles>
         <profile>
-            <id>npm-it</id>
+            <id>default</id>
```
Comment (tulioag) [inline pom.xml]: I think we should keep it for now and change in all components at the same time if that is the case. Also, I think default is not a good name for this profile.

Comment (tomivirkki) [reply pom.xml]: Changed back. I guess this should be reverted as well? https://github.com/vaadin/vaadin-board-flow/pull/110/files#diff-600376dffeb79835ede4a0b285078036L53-R53

Comment (tulioag) [reply pom.xml]: Yes, thanks.

Comment (DiegoCardoso) [reply pom.xml]: Changed back on Board as well.

File: vaadin-rich-text-editor-flow-integration-tests/pom.xml [hunk 2]
```diff
@@ -169,13 +163,6 @@
             from GitHub -->
         <!--Do not use with production mode -->
         <profile>
```
Comment (tulioag) [inline vaadin-rich-text-editor-flow-integration-tests/pom.xml]: The rest of the profile should be removed, along with the comment above.

Comment (tomivirkki) [reply vaadin-rich-text-editor-flow-integration-tests/pom.xml]: Done

Comment (tomivirkki) [reply vaadin-rich-text-editor-flow-integration-tests/pom.xml]: Done

**Pull request merged**